reporter/datadog_reporter.go

```diff
@@ -554,7 +575,7 @@ func createPprofFunctionEntry(funcMap map[funcInfo]*pprofile.Function,
 	return function
 }
 
-func addTraceLabels(labels map[string][]string, i traceFramesCounts) {
+func addTraceLabels(labels map[string][]string, i traceFramesCounts, baseExec string) {
```

Comment:
Why not have `addTraceLabels` add the "end_timestamp_ns" labels ?
This would reduce the amount of changes.